Source/WebKit/UIProcess/WebFullScreenManagerProxy.cpp

```diff
@@ -72,6 +72,11 @@ WebFullScreenManagerProxy::~WebFullScreenManagerProxy()
     callCloseCompletionHandlers();
 }
 
+const SharedPreferencesForWebProcess& WebFullScreenManagerProxy::sharedPreferencesForWebProcess() const
+{
+    return *m_page.legacyMainFrameProcess().sharedPreferencesForWebProcess();
```

Comment:
It seems we need to assert this is not null or add a `ensureSharedPreferencesForWebProcess`

Reply:
Oh, we're effectively asserting that it's not std::nullopt by using *.